TabSanity/ArrowKeyFilter.cs

```diff
@@ -104,8 +104,9 @@ public override int Exec(ref Guid pguidCmdGroup, uint nCmdID, uint nCmdexecopt,
 					case ARROW_DOWN:
 						try
 						{
-							_snapshotLine = TextView.TextSnapshot.GetLineFromPosition(
-								Caret.Position.BufferPosition.Add(CaretLine.Length - CaretColumn + 2));
+//							_snapshotLine = TextView.TextSnapshot.GetLineFromPosition(
```

Comment:
I won't be merging in commented out code. This is what version control is for. Go ahead and just delete these 2 lines.